lib/WorseReflection/Core/Inference/Resolver/BinaryExpressionResolver.php

```diff
@@ -297,7 +297,6 @@ private function addVariable(
 
         if (!in_array($operator, [
             TokenKind::DotEqualsToken,
-            TokenKind::DotToken,
```

Comment:
there may well have been a reason for this, but it's lost in time and the tests don't care so :shrug: